src/jupyter_contrib_nbextensions/nbextensions/go_to_current_running_cell/main.js

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Jupyter-Contrib Team.
+// Distributed under the terms of the Modified BSD License.
+
+// This is an extension allows you to jump to the current running cell.
+//  You can also activate this functionality automatically,
+//  i.e., your view is always scolling to the current cell.
+
+//
+// Keyboard shortcuts: Alt-I and Alt-down (works with single cells also -- this is useful!)
+// The extension is simple, create function and then register the action and shortkey separately,
+// so that user can update the shortkey according to their need.
+
+var action_follow_cell_on; // set on registration
```

Comment:
These variables should go inside the define scope to avoid polluting the global namespace

Reply:
moved to define scope